packages/xod-arduino/src/types.js

```diff
@@ -39,10 +39,22 @@ export const Liveness = EnumType('Liveness', [
   LIVENESS.SIMULATION,
 ]);
 
-export const TConfig = Model('TConfig', {
-  XOD_DEBUG: $.Boolean,
-  XOD_SIMULATION: $.Boolean,
-});
+const xodVariablesM = {
```

Comment:
Does it really need to be extracted out of `TConfig`? If so, why in `camelCase`, what’s so special in it?

Reply:
It's a subset of two models: internal `TConfig` and `XodVariables` which is used outside the package (from `xod-client`s).
I can combine `Liveness` and `XodVariables` into another one `???Config` type, but I can't get rid of this subset without code duplication. 🤔